common/static/js/src/CookiePolicyBanner.jsx

```diff
@@ -0,0 +1,4 @@
+import React from 'react';
+import CookieBanner from '@edx/cookie-policy-banner';
+
+export function CookiePolicyBanner() { return <CookieBanner /> }
```

Comment:
Do we need this wrapper? Can we not pull the component directly into the page via the ```renderReact``` method?